### PR TITLE
data: Gauthamprasath

### PR DESCRIPTION
**Pull request by Gauthamprasath**
Profile update

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6212"></a>

### CODE REVIEW

File: data/Gauthamprasath.json [hunk 1]
```diff
@@ -20,12 +21,17 @@
         }
     ],
     "links": [
+        {
+            "group": "Socials Primary",
+            "name": "LinkedIn: Let's connect",
+            "url": "https://www.linkedin.com/in/gauthamprasath",
+            "icon": "FaLinkedinIn"
```
Comment (ChinmayMhatre) [inline data/Gauthamprasath.json:28]:
```suggestion
            "icon": "FaLinkedin"
```

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]

**Review by khushalarora — APPROVED** [review]
Thank you for contributing :tada:

**Review by harshsingh32 — COMMENTED** [review]
changes reviewd

File: data/Gauthamprasath.json (hunk 1 above)
Comment (ChinmayMhatre) [reply data/Gauthamprasath.json:28]: Oh right! My bad👍🏻

**Review by sidharthviz — APPROVED** [review]

**Comment by SaraJaoude** [comment]
You have conflicts in your file, please resolve. This is usually because your fork is not up to date.  

You may wish to have a look at the following Docs and video: https://linkfree.eddiehub.io/docs/how-to-guides/editing

**Review by iamravi1 — APPROVED** [review]
Looks quite good

**Pull request closed without merging**